.pipelines/singletenancy/cilium-overlay/cilium-overlay-e2e-step-template.yaml

```diff
@@ -166,6 +171,33 @@ steps:
     name: "validatePods"
     displayName: "Validate Pods"
 
+  - script: |
+      if [ "$CILIUM_VERSION_TAG" = "cilium-nightly-pipeline" ]; then 
+        kubectl get pod -owide -n cilium-test
+        echo "wait for pod and cilium identity deletion in cilium-test namespace"
+        ns="cilium-test"
+        while true; do
+          pods=$(kubectl get pods -n $ns --no-headers=true 2>/dev/null)
+          if [[ -z "$pods" ]]; then
+            echo "No pods found"
+              break
+          fi
+          sleep 2s
+        done
+        sleep 20s
```

Comment:
Why are sleeping again after checking the deletion of pods ? For cilium `gc` ?

Reply:
Yes, need to wait for cilium gc